grouper/background/background_processor.py

```diff
@@ -147,11 +147,9 @@ def run(self):
 
                     session.commit()
 
-                stats.log_gauge("successful-background-update", 1)
-                stats.log_gauge("failed-background-update", 0)
+                stats.log_background_run(True)
```

Comment:
`success=True` here so that it's obvious what the boolean is for. `self.plugins.log_background_run`